.github/workflows/gather-data.yml

```diff
@@ -0,0 +1,67 @@
+name: Update Data
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  fetch-plugin-data:
+    name: Fetch Plugin Data
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Setup Bun
+        uses: oven-sh/setup-bun@v1
+      - name: Install dependencies
+        run: bun install
+      - name: Run script
+        run: bun run fetch-plugin-data
+      - name: Commit changes
+        run: |
+          git config user.name "${{ github.actor }}"
+          git config user.email "${{ github.actor }}@users.noreply.github.com"
+          git add release-data.json
```

Comment:
Shouldn't this be the `plugin-data.json` file? Same for the themes.

Reply:
Good spot! Fixed!